spec/controllers/registration_controller_spec.rb

```diff
@@ -284,21 +284,18 @@
   end
 
   describe '#workflow_list' do
-    it 'should handle an APO with a single default workflow' do
+    it 'should handle an APO with a single default workflow, putting the default workflow first always' do
```

Comment:
This isn't really a "single default workflow" anymore is it.  Do we still need this test?

Reply:
yeah, i guess not, basically the same as the second test now, i'll remove